src/app.py

```diff
@@ -17,6 +18,7 @@
 from lynx.common.object import Object
 from lynx.common.scene import Scene
 from lynx.common.vector import Vector
+from lynx.common.actions.create_object import CreateObject
 from pydantic import BaseModel
```

Comment:
Separate standard library, imported library and our own library imports if you have a chance